### PR TITLE
CDAP-3820 Removing distro specific code from Explore container start-up

### PR DESCRIPTION
**Pull request by poornachandra**

### CODE REVIEW

**Comment by poornachandra** [comment]
Build - http://builds.cask.co/browse/CDAP-DUT2904-1

**Comment by poornachandra** [comment]
Tested on CM CDH-5.4.7 and MapR-5.0 clusters.

**Comment by albertshau** [comment]
:+1:

File: cdap-master/src/main/java/co/cask/cdap/data/runtime/main/ExploreCustomClassLoaderTwillRunnable.java [hunk 1]
```diff
@@ -58,12 +56,12 @@ public TwillRunnableSpecification configure() {
   @Override
   public void initialize(TwillContext context) {
     super.initialize(context);
-    
+
     if (getClass().getClassLoader() instanceof URLClassLoader) {
-      LOG.info("Using CustomResourcesClassLoader to load config files bundled with application.");
+      LOG.info("Using ResourcesClassLoader to load config files bundled with application.");
```
Comment (nitinmotgi) [inline cdap-master/src/main/java/co/cask/cdap/data/runtime/main/ExploreCustomClassLoaderTwillRunnable.java]: debug ?

Comment (poornachandra) [inline cdap-master/src/main/java/co/cask/cdap/data/runtime/main/ExploreCustomClassLoaderTwillRunnable.java]: Done.

**Comment by poornachandra** [comment]
Build is green, merging.

**Pull request merged**